src/pages/ProjectView.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { H1, H2, P } from '../components/Typography'
+
+import { getSubmission } from '../utility/firebase'
+import { MAX_CHARACTERS_IN_DESCRIPTION } from '../utility/Constants'
+import { cutString, hexToRgba } from '../utility/utilities'
+import { Loading } from '../components/HeroPage'
+import Youtube from '../components/Youtube'
+import { Button } from '../components/Input'
+
+const StyledProjectContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 40px;
+  margin: 0 50px;
+`
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-width: 100%;
+`
+
+const StyledYoutube = styled(Youtube)`
+  width: 600px;
+  height: 350px;
+  border-radius: 3px;
+`
+const StyledBanner = styled.div`
```

Comment:
Can we add `text-align: center` to the banner styles?

Reply:
Added, didn't seem to change much for me tho?

Reply:
If you view it at a viewport small enough to break either the title or team member list into two lines, it should be left-aligned (which is probably not what we want) - lmk if this shows up for you!

Reply:
Oooh I see thanks!